9.4/root/usr/bin/container-entrypoint

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
```

Comment:
for sake of consistency with other script I would prefer to remove it, unless you have good reason to keep it, but then you're committing yourself to fix it everywhere :-P